### ISSUE TITLE
Multiple Battles in Same Turn

### ISSUE THREAD

**Issue by RiLeone** [RiLeone/warbot#16]
**Describe the bug**
When creating the battle pairs, if more than one pair is picked a series of things go bad, including:
- Even if only one player remains, iterations continue
- Some players end the turn being listed among the winners, but have no neighbors, e.g.
```
Surviving regions are:
	Uri: Population 75062, Area 1762 km2, Neighbors: []
```

**To Reproduce**
1. Enable multiple battle pairs by uncommenting `range(20)` at start of `for`-loop in `get_pairs()` in `src/WarBot.py`
2. Run `./WarBot.py` from `src/`
3. Check debug prints.

**Expected behavior**
Survivors should always have neighbour by definitions. If only one player remains, game should end (as long as no insurgence is allowed). 

**Additional context**
There might be more to it, but this is a start.

**Comment by SGE-MagHead** [comment]
This is the part commented out in the code right?

**Comment by RiLeone** [comment]
@SGE-MagHead yes, check the **To Reproduce** bit of the issue to see what's going on. It should be line 106 of `WarBot.py` (current code state) [Search for TODO and you should find the triggering line] 😄

**Comment by RiLeone** [comment]
This might have been solved - looks like winner states were added to all neighbourhoods, causing the graph to be linked in the wrong manner. Issue was probably solved during implementation of solution to #29. Still need to test it.

**PLEASE DO NOT WORK ON THIS ISSUE FOR THE TIME BEING**
